---
title: "Ionotropic receptors as the driving force behind human synapse establishment"
subtitle: "Supplementary Material"
author: 
  - Lucas H. Viscardi
  - Danilo O. Imparato
  - Maria CÃ¡tira Bortolini
  - Rodrigo J. S. Dalmolin
abstract: |
  The origin of nervous systems is a main theme in biology and its mechanisms are largely underlied by synaptic neurotransmission. The search for pan-neuronal genes has failed to explain synapses emergence since synaptic elements are present in multiple aneural organisms. We questioned how the interactions among these elements evolved and to what extent does it relate to our understanding of the nervous systems complexity. We infer the human neurotransmission gene network based on genes present in GABAergic, glutamatergic, serotonergic, dopaminergic, and cholinergic systems, and reconstruct the evolutionary scenario of synapse emergence. Our results outline the distribution of neurotransmitter systems and synaptic functions as taxa diverge. We find that the ionotropic receptors emergence in the cnidarian last common ancestor was the driving force behind the advent of the anatomical synapse and the development of nervous systems. We suggest that after Placozoa origin there was a positive selection under a more restricted and specific activation of ionotropic receptors. This different selective pressure counterparting the broader amino acid activation in plants could be striking for the evolution of the nervous system.
  
keywords:
  - Evolution
  - Nervous system
   
header-includes:
  # kableExtra
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{array}
  - \usepackage{multirow}
  - \usepackage{wrapfig}
  - \usepackage{float}
  - \usepackage{colortbl}
  - \usepackage{pdflscape}
  - \usepackage{tabu}
  - \usepackage{threeparttable}
  - \usepackage{threeparttablex}
  - \usepackage[normalem]{ulem}
  - \usepackage{makecell}
  - \usepackage{xcolor}
  - \usepackage{subfig}
  # Table caption left alignment
  - \usepackage{caption}
  - \captionsetup{justification=raggedright,singlelinecheck=false}
  - \captionsetup{margin={2pt,0pt}}
  # Hold figure position
  - \floatplacement{figure}{H}
  # Line numbers
  - \usepackage{lineno}
  - \linenumbers
  # Fira code
  - \setmonofont[Path=style/,Contextuals={Alternate},AutoFakeSlant]{FiraCode-Light.ttf}
  - \makeatletter
  - \def\verbatim@nolig@list{}
  - \makeatother
  # Small font size for output chunks
  - \let\oldverbatim\verbatim
  - \let\endoldverbatim\endverbatim
  - \renewenvironment{verbatim}{\scriptsize\oldverbatim}{\endoldverbatim}
  # Small font size for code chunks
  - \let\oldShaded\Shaded
  - \let\endoldShaded\endShaded
  # - \renewenvironment{Shaded}{\scriptsize\oldShaded}{\endoldShaded}
  # Rounded code blocks
  - \usepackage{tcolorbox}
  - \tcbuselibrary{breakable, skins}
  - \definecolor{chaos-bg}{HTML}{161616}
  - \newtcolorbox{blackbox}{colback=chaos-bg,frame hidden,boxsep=0pt,top=10pt,bottom=10pt,arc=6pt,enhanced jigsaw,breakable}
  - \renewenvironment{Shaded}{\scriptsize\begin{blackbox}}{\end{blackbox}}
  # emoji
  - \newfontfamily{\emojifont}[Path=style/]{seguiemj.ttf}
  - \DeclareTextFontCommand{\e}{\emojifont}
  

geometry: margin=1in

graphics: yes

output:
  pdf_document:
    toc: yes
    toc_depth: 3
    fig_caption: true
    latex_engine: xelatex
    template: style/default.tex
    pandoc_args: --highlight-style=style/chaos_modified.theme
    keep_tex: true
---
```{r setup, include=FALSE}
gc()
knitr::knit_meta(clean = TRUE)
knitr::opts_chunk$set(fig.pos = 'H')
knitr::opts_chunk$set(class.source = "numberLines")
```

<!--- Project structure -->
Project structure
=====================
This project is structured in the form of an R research compendium. A research compendium aims to provide a *"standard and easily recognizable way for organizing the digital materials of a research project to enable other researchers to inspect, reproduce, and extend the research"* [(Marwick, Boettiger &  Mullen, 2018)](https://doi.org/10.1080/00031305.2017.1375986). This project is organized as an R package, and therefore adheres to its development practices.  
The analysis itself is contained inside `analysis/` and only uses data from `data/`. This data is provided by scripts inside `data-raw/`. All analysis and preprocessing steps are documented as RMarkdown notebooks.  
Project structure:  

`â””â”€â”€`\e{ğŸ“¦} `neurotransmissionevolution/`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“‚} `analysis/`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ“} `plots/`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ““} `geneplast.Rmd`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ“„} `geneplast_roots.tsv`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â””â”€â”€` `...`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“‚} `data/`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ’¾} `cogs.rda`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ“„} `gene_annotation.tsv`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â””â”€â”€` `...`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“‚} data-`raw/`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ““} `tree_ncbi.Rmd`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ“„} `tree_ncbi.nwk`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â””â”€â”€` `...`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“} `inst/`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“} `man/`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“‚} `manuscripts/`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ““} `supplement.Rmd`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ“°} `supplement.pdf`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â””â”€â”€` `...`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“} `R/`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â”œâ”€â”€`\e{ğŸ“„} `utils.R`  
Â Â Â Â Â Â `â”‚`Â Â Â Â `â””â”€â”€` `...`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“„} `DESCRIPTION`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{ğŸ“„} `NAMESPACE`  
Â Â Â Â Â Â `â”œâ”€â”€`\e{Â®} `neurotransmissionevolution.Rproj`  
Â Â Â Â Â Â `â””â”€â”€`\e{ğŸ“„} `README.md`

<!-- Preprocessing -->
Preprocessing
=================
This topic refers mainly to data wrangling done before the actual analysis with the intent of making it simpler.

<!--- Eukaryota species tree -->
Eukaryota species tree
--------------------------
We opted to use the TimeTree database in order to obtain an standardized Eukaryota species tree. However, some species were not present in it, so we devised a way to fill them in based on NCBI Taxonomy data. 

<!-- NCBI Taxonomy tree -->
### NCBI Taxonomy tree
First we preprocess NCBI Taxonomy data to leave only STRING eukaryotes, thus making the task easier.
\input{data-raw.tree_ncbi.tex}
<!-- \input{download_files.tex} -->

<!-- Duplicated Genera -->
### Duplicated Genera
Some species from different kingdoms may share the same genus name. These genera must be noted down because one of the ways we fill in missing species is by looking at genera names.
\input{data-raw.duplicated_genera.tex}

<!-- Hybrid tree -->
### Hybrid tree
Once we have both the NCBI eukaryotes tree and the list of duplicated genera, we can start assembling the complete hybrid tree.
\input{data-raw.tree_hybrid.tex}

<!--- Kegg annotation -->
Gene selection and annotation
---------------------------------
The anchoring point for this study is basic annotation about genes and the pathways in which they participate. This section describes the process of structuring such data. In the end we will have a table to which all kinds of additional data will be left joined into.
\input{data-raw.kegg.tex}

<!--- Neuroexclusivity -->
Neuroexclusivity
--------------------
Neuroexclusivity data consists of gene expression collected from Gexe Expression Atlas and the KEGG pathways themselves.
\input{data-raw.neuroexclusivity.tex}

<!--- COG data -->
Orthology data
--------------------
This section refers to orthology data exported for geneplast use. Essentialy, we subset the global STRING mapping between proteins and orthologous groups into a smaller dataset containing only information about the orthogroups related to our selected genes.
\input{data-raw.cogs.tex}

<!--- Network -->
Network
-----------
In this section we search the STRING API for our proteins of interest and recompute the combined interaction score.
\input{data-raw.string_edgelist.tex}


<!-- Analysis -->
Analysis
============
Analysis

Root inference
---------------
To estimate the evolutionary root of a given gene, i.e. the ancestor from which its genetic archetype (orthologous group) is vertically inherited, we use orthologous group annotation from the STRING database. The presence and absence of orthologous groups in the species of a cladogram are used to determine its most likely ancestor.
\input{analysis.geneplast.tex}

<!--- Neuroexclusivity -->
Neuroexclusivity
-----------------
We characterize genes' relevance to the nervous system by inspecting what proportion of its activity is related to nervous processes. We relied on tissue RNA-Seq data, as well as KEGG's pathways themselves.
\input{analysis.neuroexclusivity.tex}

<!--- Network -->
Network
-----------
In this section we search the STRING API for our proteins of interest and recompute their combined interaction score.
\input{analysis.network.tex}

<!--- Abundance -->
Abundance
-----------
Abundance is the average number of proteins in neurotransmission orthogroups present in a species. This abundance is decomposed by orthogroup function. The orthogroup function is considered to be the function of its humans proteins in neurotransmission, as described in previous sections. In other words: "what would be the average number of "X"-related proteins in neurotransmission COGs in a species?"
\input{analysis.abundance.tex}


```{r end}
#
```
